<?xml version='1.0' encoding='UTF-8'?>
<defects4j>
	<project projectid="Mockito">
		<bug bugid="Mockito-14">
			<tests>
				<testfile
					path="org/mockitousage/bugs/VerifyingWithAnExtraCallToADifferentMockTest">
					<exception>junit.framework.AssertionFailedError:</exception>
				</testfile>
				<spectra>
					<number_tests>732</number_tests>
					<failed>1</failed>
					<passed>731</passed>
					<nodes>2922</nodes>
				</spectra>
			</tests>
			<fixlocations>
				<file path="org/mockito/internal/MockHandler.java">
					<insert numberlines="1"></insert>
					<insert numberlines="1,1">74-76,79</insert>
					<bugtypes>
						<id id="1.2" lines="19,75,78">
							<description>Incorrect case handling due to missing condition.
							</description>
						</id>
					</bugtypes>
				</file>
				<file path="org/mockito/internal/MockitoCore.java">
					<change>73</change>
					<insert numberlines="1"></insert>
					<bugtypes>
						<id id="3.1" lines="22,73">
							<description>Missing verification.</description>
						</id>
					</bugtypes>
				</file>
			</fixlocations>
			<numberfixedlines>5</numberfixedlines>
		</bug>
	</project>
</defects4j>
